docs/webhooks.rst

```diff
@@ -51,7 +52,7 @@ GitHub
 .. note:: The webhook secret is not yet respected
 
 Bitbucket
-~~~~~~~~~
```

Comment:
This alters the nesting negatively. We want to keep this as it was to keep `Bitbucket` under `Webhook Integrations`